doc/generated/pmemobj_ctl_get.3

```diff
@@ -456,8 +456,8 @@ contains a sequence of ctl queries.
 The parsing rules are all the same, but the file can also contain
 white\-spaces and comments.
 .PP
-To create a comment, simply use \[aq]#\[aq] anywhere in a line and
-everything afterwards, until a new line \[aq]\[aq], will be ignored.
+To create a comment, simply use `#' anywhere in a line and everything
+afterwards, until a new line `', will be ignored.
```

Comment:
It should be  something like that: `'\n'`,

Reply:
Also, it replaces ```""``` with ``` `' ``` etc.